blockchain/blockchain-wallet-v4-frontend
# Refactor Unused & Used Addresses

## Description
Refactors and then finishes previously missed features for unused and used addresses page.
- Adds the ability to view previously used addresses of wallets
- Moves most actions logic to component sagas
- Fixes logic issues with determining if an address is used vs unused
- Adds loading and empty states to tables
- Fixes issue where unused address deletion didnt update table automatically
- Adds filtering to both used and unused addresses tables via the search box in sub menubar

## Change Type
Feature/Bug Fix

## Testing Steps
Log into an HD wallet with previous receive transactions and verify the old addresses show.
Log into an HD wallet and create new unused addresses. Ensure ability to edit and delete these addresses works.

## PR Creator Checklist

## PR Reviewer Checklist
